src/ServiceControl.Audit.Persistence.RavenDb/ServiceControl.Audit.Persistence.RavenDb.csproj

```diff
@@ -42,8 +42,12 @@
   </ItemGroup>
 
   <Target Name="CopyRavenStudio" BeforeTargets="AssignTargetPaths">
+    <PropertyGroup>
+      <PackageReferenceIdentity>%(PackageReference.Identity)</PackageReferenceIdentity>
+      <PackageReferenceIdentityLower>$(PackageReferenceIdentity.ToLower())</PackageReferenceIdentityLower>
+    </PropertyGroup>
     <ItemGroup>
-      <None Include="$(NuGetPackageRoot)%(PackageReference.Identity)\%(PackageReference.Version)\tools\Raven.Studio.Html5.zip" CopyToOutputDirectory="PreserveNewest" Condition="'%(PackageReference.Identity)' == 'RavenDB.Database'" />
+      <None Include="$(NuGetPackageRoot)$(PackageReferenceIdentityLower)\%(PackageReference.Version)\tools\Raven.Studio.Html5.zip" CopyToOutputDirectory="PreserveNewest" Condition="'$(PackageReferenceIdentityLower)' == 'ravendb.database'" />
```

Comment:
:nauseated_face: if there's a better way to do this, I'd love to know about it.

Reply:
@adamralph There is, see #3127